clang/lib/DPCT/APINames_PyTorch.inc

```diff
@@ -0,0 +1,39 @@
+//===------------------------ APINames_PyTorch.inc ------------------------===//
```

Comment:
I think there is no need to introduce this file. pytorch framework API is not cuda API.

Reply:
Okay, I'll remove this file.